Assets/Scripts/Game/TransportManager.cs

```diff
@@ -60,6 +60,7 @@ public PlayerPositionData_v1 BoardShipPosition
         }
 
         public ImageData RidingTexture { get { return ridingTexture; } }
+        public int FrameIdx { get { return frameIdx; } }
```

Comment:
I'm not a fan of how `index` was abbreviated to `idx` (only two chars saved and is somewhat less readable) but this is just personal preference.

Reply:
Oh, okay. I've abbreviated Index to Idx so much I find it just as readable. Happy to change it here as readability is important.